include/internal/libspdm_responder_lib.h

```diff
@@ -240,6 +240,29 @@ libspdm_return_t libspdm_get_response_measurements(libspdm_context_t *spdm_conte
                                                    const void *request,
                                                    size_t *response_size,
                                                    void *response);
+
+/**
+ * Process the SPDM GET_MEASUREMENT_EXTENSION_LOG request and return the response.
+ *
+ * @param  spdm_context                  A pointer to the SPDM context.
+ * @param  request_size                  size in bytes of the request data.
+ * @param  request                      A pointer to the request data.
+ * @param  response_size                 size in bytes of the response data.
+ *                                     On input, it means the size in bytes of response data buffer.
+ *                                     On output, it means the size in bytes of copied response data buffer if RETURN_SUCCESS is returned,
+ *                                     and means the size in bytes of desired response data buffer if RETURN_BUFFER_TOO_SMALL is returned.
+ * @param  response                     A pointer to the response data.
+ *
+ * @retval RETURN_SUCCESS               The request is processed and the response is returned.
+ * @retval RETURN_BUFFER_TOO_SMALL      The buffer is too small to hold the data.
+ * @retval RETURN_DEVICE_ERROR          A device error occurs when communicates with the device.
+ * @retval RETURN_SECURITY_VIOLATION    Any verification fails.
+ **/
+libspdm_return_t libspdm_get_response_measurement_extension_log(libspdm_context_t *spdm_context,
```

Comment:
MEL is its own capability. This should be within `LIBSPDM_ENABLE_CAPABILITY_MEL_CAP`.